CHANGELOG

```diff
@@ -1,3 +1,33 @@
+v4.0.0.pre6 (33 commits)
+------------------------------
+
+Improvements (4)
+----------------
+
+* Create borderless windows [f85f8f2]
+* Lighten the load on Travis a bit [9ab21f5]
```

Comment:
You're right... must have gotten tagged as an improvement in the commit message and showed up in the automated list from `rake changelog`. Fixed!